src/backends/tensorrt/plan_backend.cc

```diff
@@ -170,6 +170,12 @@ PlanBackend::Context::Context(
     events_[idx].timestamp_signal_ = nullptr;
   }
   support_batching_ = (max_batch_size != NO_BATCHING);
+
+  Status status = SupportsIntegratedZeroCopy(gpu_device_, &zero_copy_support_);
+  if (!status.IsOk()) {
+    LOG_ERROR << "Failed to check if zero copy is supported";
```

Comment:
I think it might be useful to know why zero copy is not supported. That is why I retained the reason in the log.

Reply:
I don't think you have printed the reason in log. By the way, it may be better to mention the optimization for zero copy will be disabled (not a fatal error, just that the performance may be less optimized)